include/fmt/chrono.h

```diff
@@ -336,8 +336,6 @@ template <typename CodeUnit> struct codecvt_result {
   CodeUnit buf[max_size];
   CodeUnit* end;
 };
-template <typename CodeUnit>
-constexpr const size_t codecvt_result<CodeUnit>::max_size;
```

Comment:
Suggest:
```c++
#if FMT_CPLUSPLUS < 201703L
template <typename CodeUnit>
constexpr const size_t codecvt_result<CodeUnit>::max_size;
#endif
```

Reply:
I'd prefer avoiding conditional compilation if possible.

Reply:
@vitaut 
This is C++ :(

https://en.cppreference.com/w/cpp/language/static#Static_data_members

> If a const _non-inline (since C++17)_ static data member _or a constexpr static data member (since C++11)(until C++17)_ is odr-used, a definition at namespace scope is still required, but it cannot have an initializer.

Reply:
But it doesn't need to be ODR-used so it should be fine.